### ISSUE TITLE
Set/shift for double function buttons

### ISSUE THREAD

**Issue by BeyondBuka** [raybellis/emu.tools-public#9]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Set button on the emu is used as a shift button on the keyboard. E-remote could implement this by making the shift button stay pressed on double tap or something so it could be used in combination with other buttons because with the mouse you cant press two buttons at once.

**Describe alternatives you've considered**

**Additional context**

**Comment by raybellis** [comment]
Yeah, I've been wondering how to implement that one.  I could also perhaps use the keyboard shift-key to send shift/set.

**Comment by BeyondBuka** [comment]
> Yeah, I've been wondering how to implement that one. I could also perhaps use the keyboard shift-key to send shift/set.

Great mate, thanx!
